### DISCUSSION TITLE
Push notification to iPhone if door bell button is pressed

### DISCUSSION THREAD

**Discussion by etkaar** [HomeSpan/HomeSpan#864] [Q&A]
How can I send a push notification to the iPhone once a button such as a door bell button is pressed? I had a look to the Reference.md but have not found something.

**Comment by HomeSpan** [comment]
HomeKit has a specific service for a DoorBell that allows it to ring on a HomePod automatically.  Use the Doorbell Service with the ProgrammableSwitchEvent Characteristic.
